ci-operator/jobs/openshift/release/openshift-release-master-postsubmits.yaml

```diff
@@ -275,6 +275,8 @@ postsubmits:
         - --config-dir=core-services
         - --config-dir=clusters/build-clusters/common
         - --confirm=true
+        - --kubeconfig
```

Comment:
nit: the rest of the options are single lines `- --flag=value` - can we do that here, too?

Reply:
Updated.